maplibre-adapter/src/main/java/com/amplifyframework/geo/maplibre/view/MapLibreView.kt

```diff
@@ -72,10 +83,15 @@ class MapLibreView
     }
 
     lateinit var symbolManager: SymbolManager
+    internal lateinit var symbolOnClickListener: (Symbol) -> Boolean
 
     var defaultPlaceIcon = R.drawable.place
     var defaultPlaceActiveIcon = R.drawable.place_active
 
+    private var shouldCluster = true
```

Comment:
Could these be lateinit?

Reply:
These 3 properties are initialized to their default values, which I think makes the code a bit cleaner compared to using lateinit.